internal/services/eventhub/eventhub_namespace_customer_managed_key_resource.go

```diff
@@ -84,6 +86,8 @@ func resourceEventHubNamespaceCustomerManagedKey() *pluginsdk.Resource {
 				Default:  false,
 				ForceNew: true,
 			},
+
+			"identity": commonschema.SystemOrUserAssignedIdentityOptional(),
```

Comment:
so i think this needs to be "per key vault meaning the above would be marked as deprecated and we should create a new block:

```
"key_vault": {
	Type:     pluginsdk.TypeSet,
	Optional: true,
	AtLeastOneOf: key_vault, key_vault_key_ids
	Elem: &pluginsdk.Schema{
		"key_vault_key_ids": {
				Type:     pluginsdk.TypeSet,
				Required: true,
				Elem: &pluginsdk.Schema{
					Type:         pluginsdk.TypeString,
					ValidateFunc: keyVaultValidate.NestedItemIdWithOptionalVersion,
				},
			},

		"identity": commonschema.SystemOrUserAssignedIdentityOptional(),

}
```